subprojects/core/src/main/java/org/gradle/configuration/project/LifecycleProjectEvaluator.java

```diff
@@ -76,7 +77,13 @@ private void notifyAfterEvaluate(ProjectEvaluationListener listener, ProjectInte
         } catch (Exception e) {
             if (state.hasFailure()) {
                 // Just log this failure, and pass the existing failure out in the project state
-                LOGGER.error("Failed to notify ProjectEvaluationListener.afterEvaluate(), but primary configuration failure takes precedence.", e);
+                boolean logStackTraces = project.getGradle().getStartParameter().getShowStacktrace() != ShowStacktrace.INTERNAL_EXCEPTIONS;
+                String infoMessage = "Failed to notify ProjectEvaluationListener.afterEvaluate(), but primary configuration failure takes precedence.";
```

Comment:
❌ We should reword this too, because users mistake this for the real error. How about:

`Project evaluation failed, leading to a follow-up error in Project.afterEvaluate {}. Run with --stacktrace to see the follow-up error`.